1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,9 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function() {
+        return this.age ++;
```

Comment:
My recommendation is not to use space before `this.age++`. I've never seen it at work or uni using space